File: ASP.NETCoreWebApplication/Models/VagtModel.cs
using System;
using System.Collections.Generic;
using ASP.NETCoreWebApplication.Entities;

namespace ASP.NETCoreWebApplication.Models
{
    public class VagtModel
    {
        public int VagtId { get; set;}
        public  DateTime Start { get; set; }
        public DateTime End { get; set; }
        public Medarbejder Medarbejder { get; set; }
        public virtual ICollection<Opgave> Opgaver { get; set; }
    }
}

File: ASP.NETCoreWebApplication/ClientApp/src/app/Models/vagt.model.ts
import {MedarbejderModel} from "./medarbejder.model";
import {OpgaveModel} from "./opgave.model";

export class VagtModel {
  vagtID: number;
  start: Date;
  end: Date;
  medarbejder: MedarbejderModel;
  opgaver: Array<OpgaveModel> = [];
}
